### ISSUE TITLE
linter: `eslint/capitalized-comments` fails with options configuration

### ISSUE THREAD

**Issue by jeanne-cai** [oxc-project/oxc#23122]
### What version of Oxlint are you using?

1.69.0

### What command did you run?

`yarn oxlint -c oxlint.config.mjs`

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

```ts
import { defineConfig } from 'oxlint';

export default defineConfig({
    plugins: ['eslint'],
    rules: {
        'eslint/capitalized-comments': [
            'error',
            'always',
            {
                ignoreConsecutiveComments: true,
            },
        ],
    },
});
```

### What happened?

Providing options to `eslint/capitalized-comments` triggers a configuration error:

```bash
Failed to parse oxlint configuration file.

  × Invalid configuration for rule `capitalized-comments`:
  │   This rule does not accept configuration options.
```
